T-Editing-Guidelines-201602

anermina opened the pull request:
Converted T-Editing-Guidelines-201602 document

anermina committed:
T-Editing-Guidelines-201602

Converted T-Editing-Guidelines-201602 document

ronaldtse commented on sources/T-Editing-Guidelines-201602.adoc:
```diff
@@ -0,0 +1,843 @@
+= Author's guide for drafting ITU-T Recommendations
+ITU
```
@anermina why is there an "ITU" line here? Let's remove it.

anermina replied:
I'm not sure why this was written in G.650 and Y.3507 documents, which I used as a template. Will remove this line.

ronaldtse replied:
Thanks, we should also remove it from those two files.

opoudjis replied:
_Do not. Do not remove it. Do not._

Random things break in the processing of Asciidoctor document attributes if you leave out the author name. I do not remember what they are, but there is no reason to risk them breaking just because you don't think it looks neat. 

@manuel489: this also applies to the templates you are editing. 

I'm editing https://www.metanorma.com/author/topics/document-format/meta-attributes/ to reflect this.

ronaldtse replied:
@opoudjis Unfortunately you're wrong here. The author name is only optional in AsciiDoc. There is no issue with leaving out the author name. And in this case, the name is wrong anyway. If you have been keeping up with the sample documents and templates -- ALL mn-samples-* documents DO NOT HAVE THIS FIELD. Gosh. Remove this already.

ronaldtse replied:
@anermina @manuel489 please remove these lines because they are wrong and misleading. Thanks.

ronaldtse commented on sources/T-Editing-Guidelines-201602.adoc:
```diff
@@ -0,0 +1,843 @@
+:docnumber: EDITING GUIDELINES
+:series:
+:series1:
+:series2: 
```
Do we need these three lines of `:seriesX:`? If not we can remove them.

anermina replied:
Removing

opoudjis replied:
Yes, remove them from the Editing Guidelines, but only because the Editing Guidelines don't have associated series.

ronaldtse commented on sources/T-Editing-Guidelines-201602.adoc:
```diff
@@ -0,0 +1,843 @@
+==== implementers' guide <<b-ITU-T_A.1>> 
+An implementers' guide is a document which records all identified defects (e.g., typographical errors, editorial errors, ambiguities, omissions or inconsistencies and technical errors) associated with a Recommendation or a set of Recommendations and their status of correction, from their identification to final resolution. An implementers' guide is issued by ITU-T following agreement by a study g ...
+
+[[a_3_2Terms_defined_in_this_Recommendation]]
```
@anermina we should rename all the anchors to something that non-machines can understand...

anermina replied:
I didn't understand the request well, can you please give an example?

b-ITU-T_A.1 is the name of the reference shown in the original paper, while a_3_2Terms_defined_in_this_Recommendation follows the principle from G.650 and Y.3507 - a_sectionNumber sectionTitle

Do you mean that title should be added for example here: [[a_3_1_6]] as well?

ronaldtse replied:
@anermina I think instead of `a_3_2Terms_defined_in_this_Recommendation`, the anchor could be just `terms`.

opoudjis replied:
Note that the anchors were mechanically assigned by reverse_adoc, based on titles. It is not a necessity to simply those anchors to more brief anchors (such as someone would use if they were writing the document from scratch); but since these are exemplars of ITU markup, I guess that's a harmless request.

@ronaldtse, your request here for short anchors would indeed need to be applied to all the ITU sample docs; and since Manuel is currently working on templates, that would end up being a task for @anermina.

ronaldtse replied:
@anermina I meant something like this. Notice the reference is actually a `[.source]` element. It is only rendered next to the term in presentation. In semantic markup we do not want to mix concepts.

```adoc

[[terms-implementers-guide]]
==== implementers' guide
An implementers' guide is a document which records all identified defects (e.g., typographical errors, editorial errors, ambiguities, omissions or inconsistencies and technical ...

[.source]
<<b-ITU-T_A.1>> 
```

ronaldtse replied:
I've fixed this in 142b85d.

anermina replied:
Sorry, I didn't manage to finish this earlier. I will fix the rest of the cross-references in this document (such as <<a_7Description_of_introductory_material_to_be_provided_by_the_author>>), if it is ok. I've already started doing the same for G.650 document and can do it for the rest of the ITU-T documents as well.

ronaldtse replied:
@anermina that's great, please proceed.

ronaldtse replied:
Please remember to run `git pull` to fetch the latest master. Thanks.

anermina replied:
I've made a new pull request for this document: #41

ronaldtse commented on sources/T-Editing-Guidelines-201602.adoc:
```diff
@@ -0,0 +1,843 @@
+* [[[b-ITU-T_K.35,b-ITU-T K.35]]], Recommendation ITU-T K.35 (1996), _Bonding configurations and earthing at remote electronic sites_.
+
+* [[[b-ITU-T_Rap,b-ITU-T Rap]]], ITU-T, _Manual for Rapporteurs and Editors_, 12 February 2010. +
+http://www.itu.int/oth/T0A0F000006/en[http://www.itu.int/oth/T0A0F000006/en]
```
I think we probably don't want manual formatting or line breaks with references. @opoudjis your two cents please.

opoudjis replied:
Nermina's brief was to replicate source markup to the extent possible, so that we don't raise questions about what Metanorma can't do. I am normally as vehemently opposed as you are to capricious format; in my opinion though this instance is harmless, and I'd prefer it to stay.

ronaldtse replied:
@opoudjis Metanorma does things semantically. Non-semantic rendering should be either removed or made semantic.

ronaldtse commented on sources/T-Editing-Guidelines-201602.adoc:
```diff
@@ -0,0 +1,843 @@
+
+The draft <title of the draft>, which is proposed for approval, consent or determination:
+
+[style=circle]
```
I don't think we should be changing the list styles here, the priority is to keep things for their meaning, not their presentation. (which is the intention of ITU)

anermina replied:
Removing [style=circle] and [circle]

opoudjis replied:
As it happens, there are styling commands which Metanorma completely ignores anyway, and choice of bullet is one of them. (And as we just saw with steps, the only way we will allow a bullet variant is if there is already a defined CSS style in the Word template -- meaning, as a CSS class of lists. We probably can apply Word CSS for bullet variants inline, on an ad hoc basis (parsing "style"); but that goes against our design philosophy of constraining rendering to semantically meaningful distinctions.

ronaldtse reviewed (changes requested):
@anermina could you help handle the comments stated? Thanks!

anermina committed:
Update T-Editing-Guidelines-201602.adoc

- Removed "ITU", :seriesX:, [style=circle], [circle]
- Formatted and centered URLs 
- Changed three tables into examples

ronaldtse committed:
Remove non-human anchors and fix terms
